Jack/Gen.fs

```diff
@@ -17,8 +17,8 @@ module Gen =
     let create (shrink : 'a -> LazyList<'a>) (random : Random<'a>) : Gen<'a> =
         Random.map (Tree.unfold id shrink) random |> ofRandom
 
-    let result (x : 'a) : Gen<'a> =
-        Tree.result x |> Random.result |> ofRandom
+    let constant (x : 'a) : Gen<'a> =
+        Tree.singleton x |> Random.constant |> ofRandom
```

Comment:
Yeah, so I like `singleton` for the `Tree`, but I'm not sure about the other ones yet. My thinking was that if it can't be the "right" word (pure or return) then I should make it something which makes sense for the given concept. For `Gen`/`Random` I think it means `constant`.